pkg/registry/servicecatalog/instance/strategy.go

```diff
@@ -19,6 +19,7 @@ package instance
 // this was copied from where else and edited to fit our objects
 
 import (
+	//apiequality "k8s.io/apimachinery/pkg/api/equality"
```

Comment:
I'm a little surprised that one of the many nag-tools we run on this repo didn't barf on this

Reply:
can you add a comment to explain why this is commented out?

Reply:
I have uncommented the code that compares the old and new spec, even though it will always evaluate to true.